### PR TITLE
fix: tool command cannot handle multi arg commands

### PR DESCRIPTION
**Pull request by sqin2019**

### CODE REVIEW

File: pkg/handler/tool_handler.go [hunk 1]
```diff
@@ -93,9 +96,26 @@ func (h *ToolHandler) run(tool string, cmds []string) error {
...
 		// Empty line in between commands.
-		if i < (len(cmds) - 1) {
+		if h.stdout != nil && i < (len(cmds)-1) {
 			fmt.Fprintf(cmd.Stdout, "\n")
 		}
 	}
 	return nil
 }
+
+func splitArgs(str string) []string {
```
Comment (yolocs) [inline pkg/handler/tool_handler.go:106]: As we discussed offline, we can try run `bash -c "gcloud xxx"` instead. Like you said, that's likely safer

File: pkg/handler/tool_handler.go [hunk 2]
```diff
@@ -84,38 +82,21 @@ func (h *ToolHandler) Cleanup(ctx context.Context, r *v1alpha1.ToolRequest) erro
 
 func (h *ToolHandler) run(tool string, cmds []string) error {
 	for i, c := range cmds {
-		args := splitArgs(c)
-		cmd := exec.Command(tool, args...)
+		script := fmt.Sprintf("%s %s", tool, c)
+		cmd := exec.Command("bash", "-c", script)
```
Comment (yolocs) [inline pkg/handler/tool_handler.go:86]: Comment here and in the CLI desc that tool only works where bash is available.

File: pkg/handler/tool_handler.go [hunk 3]
```diff
@@ -84,38 +82,21 @@ func (h *ToolHandler) Cleanup(ctx context.Context, r *v1alpha1.ToolRequest) erro
 
 func (h *ToolHandler) run(tool string, cmds []string) error {
 	for i, c := range cmds {
-		args := splitArgs(c)
-		cmd := exec.Command(tool, args...)
+		script := fmt.Sprintf("%s %s", tool, c)
```
Comment (yolocs) [inline pkg/handler/tool_handler.go:85]:
```suggestion
		toolCmd := fmt.Sprintf("%s %s", tool, c)
```

File: pkg/handler/tool_handler.go (hunk 2 above)
Comment (yolocs) [reply pkg/handler/tool_handler.go:86]: Also comment here about why we use bash.

**Review by yolocs — COMMENTED** [review]
nit

File: pkg/handler/tool_handler.go (hunk 2 above)
Comment (sqin2019) [reply pkg/handler/tool_handler.go:86]: ack

**Review by yolocs — APPROVED** [review]

**Pull request merged**